CHANGELOG.md

```diff
@@ -1,10 +1,24 @@
 ## 2.3.0
 
+### Breaking changes
+
+* `RealmResults.distinct()` returns a new `RealmResults` object instead of filtering on the original object (#2947).
```

Comment:
Thanks @Zhuinden, I didn't remember we categorized it as a bug. In that regard, fixing it here seems fine.